src/main/java/uk/gov/hmcts/reform/pip/data/management/controllers/PublicationController.java

```diff
@@ -132,4 +137,25 @@ private void validateRequestHeaders(String provenance, String sourceArtefactId)
             throw new EmptyRequestHeaderException(PublicationConfiguration.SOURCE_ARTEFACT_ID_HEADER);
         }
     }
+
+
+    /**
+     * Enforces conditional mandatory fields based on the publication type. This is to ensure that status updates are
+     * able to persist indefinitely if required.
+     */
+    public boolean validateDateFromDateTo(LocalDateTime displayFrom, LocalDateTime displayTo, ArtefactType type) {
+        if (type.equals(ArtefactType.LIST) || type.equals(ArtefactType.JUDGEMENT)
+            || type.equals(ArtefactType.OUTCOME)) {
+            if (displayFrom == null) {
+                throw new DateHeaderValidationException("Date from field is mandatory for this artefact type");
+            }
+            if (displayTo == null) {
+                throw new DateHeaderValidationException("Date to field is mandatory for this artefact type");
+            }
+        } else {
+            return !type.equals(ArtefactType.STATUS_UPDATES) || displayFrom != null;
+        }
+
+        return true;
+    }
```

Comment:
maybe in the service rather than controller?

Reply:
👍